test/stub/service.go

```diff
@@ -0,0 +1,79 @@
+package stub
+
+import (
+	"context"
+	"crypto/rsa"
+	"github.com/fabric8-services/fabric8-common/convert/ptr"
+	"github.com/fabric8-services/fabric8-tenant/auth"
+	authclient "github.com/fabric8-services/fabric8-tenant/auth/client"
+	"github.com/fabric8-services/fabric8-tenant/cluster"
+	"github.com/satori/go.uuid"
+)
+
+type ClusterService struct {
+	APIURL string
+	User   string
+	Token  string
+}
+
+func (s *ClusterService) Start() error {
+	return nil
+}
+
+func (s *ClusterService) GetCluster(ctx context.Context, target string) (cluster.Cluster, error) {
+	return cluster.Cluster{
+		APIURL: s.APIURL,
+		User:   s.User,
+		Token:  s.Token,
+	}, nil
+}
+
+func (s *ClusterService) GetClusters(ctx context.Context) []cluster.Cluster {
+	cl, _ := s.GetCluster(ctx, "")
+	return []cluster.Cluster{cl}
+
+}
+
+func (s *ClusterService) Stop() {
+}
+
+type AuthService struct {
```

Comment:
can't we introduce minimock for AuthService? It will help us to modify this stub each time for every single change of `AuthService`.

Reply:
ok